docs/gadgets/prometheus.md

```diff
@@ -194,6 +194,34 @@ metrics:
       - "status:CLOSE_WAIT"
 ```
 
+### Histograms
+
+"A _histogram_ samples observations (usually things like request durations or response sizes) and counts them in configurable buckets. It also provides a sum of all observed values." from [https://prometheus.io/docs/concepts/metric_types/#histogram](https://prometheus.io/docs/concepts/metric_types/#histogram). We support the same bucket configuration as described in
+[https://github.com/cloudflare/ebpf_exporter#histograms.](https://github.com/cloudflare/ebpf_exporter#histograms.)
+
+Right now only trace gadgets are supported.
+
+Example of histograms is:
+
+Latency of DNS requests for all pods
+
+```yaml
+metrics_name: metrics_name
+metrics:
+  - name: dns_requests_latency
+    type: histogram
+    category: trace
+    field: latency
+    bucket:
+      min: 0
+      max: 10
+      multiplier: 100000 # 0.1ms
+      type: exp2
+      unit: ns
+    selector:
+      - "qr:R" # Latency is only calculated for response events
+```
+
 ### Guide
```

Comment:
Put some instructions to run the monitoring stack you created in the last commit?

Reply:
Good Idea. Done, I added the grafana section!